src/main/java/uk/gov/companieshouse/insolvency/delta/serialization/ChsDeltaDeserializer.java

```diff
@@ -17,7 +17,6 @@
     @Override
     public ChsDelta deserialize(String topic, byte[] data) {
```

Comment:
Can we please write a unit test for this deserializer class to ensure Avro message is successfully de-serialised into ChsDelta object?